### DISCUSSION TITLE
enable use a 1-click retireval step by default - URL scanners eating up views

### DISCUSSION THREAD

**Discussion by nmuthyala7** [pglombardo/PasswordPusher#2243] [Q&A] [answered]
Hi,

Can you please enable "Use a 1-click retrieval step" checkbox by  default in the image, the URL scanners are eating up views, right now we need to check that box manually when we are creating a new secret but a lot of users are not following it, would be great to have that by default, we are hosting it on a azure web app - container deploy. Checked to see if its an env variable but doesn't look like.

Thank you

**Comment by pglombardo** [comment, marked as answer]
Hi @nmuthyala7 - add this environment variable to your container image and it will be checked by default:`PWP__PW__RETRIEVAL_STEP_DEFAULT=true`.

If you are using a `settings.yml` file, it can be enabled  there instead.

Documentation on these options [are here](https://github.com/pglombardo/PasswordPusher/blob/master/Configuration.md#password-push-expiration-settings).

Let me know if this works out for you.

**Reply by nmuthyala7** [reply]
This worked, thank you.
